VMAccess/vmaccess.py

```diff
@@ -95,10 +96,12 @@ def enable():
             _backup_sshd_config(SshdConfigPath)
 
         if reset_ssh:
+            waagent.AddExtensionEvent(name=hutil.get_name(), op="scenario", isSuccess=True, message="reset-ssh")
             _reset_sshd_config(SshdConfigPath)
             hutil.log("Succeeded in reset sshd_config.")
 
         if remove_user:
+            waagent.AddExtensionEvent(name=hutil.get_name(), op="scenario", isSuccess=True, message="remove-user")
```

Comment:
These telemetry events are just telling you the scenario, but there does not appear to be any real success/failure.  I.e. isSuccess is always True.  Perhaps you should consider including that.

Perhaps you need a telemetry event in the exception handler portion to flag a failure?